Sources/TuistKit/Services/ScaffoldService.swift

```diff
@@ -13,14 +14,17 @@ enum ScaffoldServiceError: FatalError, Equatable {
         }
     }
 
-    case templateNotFound(String)
+    case templateNotFound(String, searchPaths: [AbsolutePath])
     case nonEmptyDirectory(AbsolutePath)
     case attributeNotProvided(String)
 
     var description: String {
         switch self {
-        case let .templateNotFound(template):
-            return "Could not find template \(template). Make sure it exists at Tuist/Templates/\(template)"
+        case let .templateNotFound(template, searchPaths):
```

Comment:
current output can get pretty long if you have a lot of different templates - which could doubly be true for a plugin.

rather than having a list of `path/Templates/template_one`, `path/Templates/template_two`, etc., I think it'd be more valuable to have only the common parent directory. 

That is if I have a plugin and local templates, I'd like to see that tuist searched for them in:
`path-to-local-templates/Templates` and `path-to-plugin/Templates`.

We could probably do this by something like: 
```swift
let searchDirectories: Set<AbsolutePath> = searchPaths.reduce(into: []) { acc, path in  
   acc.insert(path.parentDirectory)
}
```

I believe this might be more helpful to the users (but correct me if you don't feel this way!)